CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # CHANGELOG
 
 ## Not released
+
 - Pass actions and options to WrapperWidgetUI from HistogramWidget and CategoryWidget [#50](https://github.com/CartoDB/carto-react-lib/pull/50)
-- Fix addSource in cartoSlice spreading layerAttributes [#52∫](https://github.com/CartoDB/carto-react-lib/pull/52)
+- Fix addSource in cartoSlice spreading layerAttributes [#52](https://github.com/CartoDB/carto-react-lib/pull/52)
+- extend widget props to WrapperWifgetUI with a better implementation than in [#50](https://github.com/CartoDB/carto-react-lib/pull/50) [#57](https://github.com/CartoDB/carto-react-lib/pull/57)
 
 ## 1.0.0-beta8 (2020-12-15)
+
```

Comment:
We didn't apply here a linter on markdown, so maybe remove the blanks, don't you think?

Reply:
This blanks were added by my editor (VS Code) automatically with Prettier "format on save" plugin, but I can remove them if you want

Reply:
Ok, let them in. We'll try to add prettier to the project, as we had before on carto-react-template